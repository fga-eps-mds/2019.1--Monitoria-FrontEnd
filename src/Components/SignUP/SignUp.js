import React, { Component } from 'react';
import { Grid, Button, TextField } from '@material-ui/core' ;
import logo from '../../Assets/img/Logo.png';
import './SignUp.css';



class SignUp extends Component {
  render() {
    return (
      <div className="SignUpBackground">
        <Grid container alignContent="center" justify="center" direction="column" alignItems="center">
          <img src={logo} alt="Logo" width="120" height="120"/>
          <Grid item >
            <TextField
            id="nomeTextField"
            label="Nome"
            margin="normal"
            />

          </Grid>
          <Grid item >
            <TextField
              id="emailTextField"
              label="Email"
              margin="normal"
              />
          </Grid>
          <Grid item >
            <TextField
              id="senhaTextField"
              label="Senha"
              margin="normal"
              type="password"
             
              />
          </Grid>
          <Grid item >
            <TextField
              id="repetirSenhaTextField"
              label="Repetir senha"
              margin="normal"
              type="password"
             
              />
          </Grid>
          </Grid>
          <Grid container alignContent="center" justify="center" direction="column" spacing="24" alignItems="center" style={{marginTop: 25}}>
          <Grid item >
              <Button variant="outlined">
                Registrar
              </Button>
              </Grid>
            <Grid item>
              <Button variant="outlined" >
                Cancelar
              </Button>
            </Grid>
      
        </Grid>
            
            
        </div>
    );   
  }
}

export default SignUp;