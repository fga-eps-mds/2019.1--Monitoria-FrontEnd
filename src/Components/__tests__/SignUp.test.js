import React from 'react';
import SignUp from '../SignUP/SignUp';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('Testing SignUp component', () => {
    it('Test if SignUp renders correctly', () =>{
        const tree = shallow(<SignUp/>);
        expect(toJson(tree)).toMatchSnapshot();
    });
});
