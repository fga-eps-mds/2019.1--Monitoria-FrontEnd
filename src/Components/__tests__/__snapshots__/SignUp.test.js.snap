// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing SignUp component Test if SignUp renders correctly 1`] = `
<Route>
  <Component />
</Route>
`;
